packages/client/src/server.ts

```diff
@@ -29,6 +30,37 @@ import {
 } from './omega_edit_pb'
 import { runServer } from '@omega-edit/server'
 
+/**
+ * Wait for pidfile to be created
+ * @param pidfile reolved path to the pidfile
+ */
+export async function pidfileExists(pidfile: string) {
+  return new Promise((resolve, reject) => {
+    var timer = setTimeout(function () {
+      watcher.close()
+      reject(new Error('PID file not created after 1 seconds'))
```

Comment:
1000ms == 1s

Reply:
The comment was about seconds in plural.  Timeouts are in milliseconds, so take your pick.  I prefer ms since that is the measurement being used in `setTimeout` and if we make the timeout time a function parameter, we'd want to use `ms` instead of `sec`.